operations/customers.md

```diff
@@ -225,6 +230,11 @@ Returns all customers filtered by identifiers, emails, names and other filters.
 * `SendMarketingEmails`
 * ...
 
+#### Activity State
```

Comment:
The activity state is already defined in Vouchers object, might be worth to extract it to separate page wit h such a common elements.

Reply:
What might be the best page for that? just reference it from vouchers? I think if we don't have such a page yet typically we have duplicated objects... same for update values, etc.

Reply:
Current policy on this question is as follows:
1. Where there are common data objects referenced in multiple places, define the object once close to where it is first used or most frequently used and link to that one instance, even if on a separate page
2. EXCEPT... for very minor data objects (like Activity State or Update Value), these can be duplicated on all the pages where they are referenced
3. Regardless, a data object should never be defined more than once on the same page; and if multiple definitions have the same name, then they must have the same definition!
4. In general, being consistent is more important than the actual method used

Of course the policy can be changed, but that is the current policy. So in this case, it's fine to define Activity State on this page as long as (1) it is only defined once on the page; (2) the definition is exactly the same as appears in vouchers.md; and (3) for consistency, all instances of references to Activity State on this page should link to the same definition (i.e. #activity-state).

Phew!